### PR TITLE
Update README and add PR template

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** assorted documentation/GitHub updates

JIRA: n/a

**What changed?**

- Add @edx/masters-devs as CODEOWNERs
- Some README cleanup:
  - Add missing info about `Bulk Management` feature flag
  - Remove old Neem team tag
  - Add some jump links
- Add VS Code settings to .gitignore
- Add PR template

**Developer Checklist**
- [x] Test suites passing
- [x] Received code-owner approving review
- [ ] ~Bumped version number [package.json](../package.json)~ Will not do, just edits docs

**Testing Instructions**

n/a

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by muselesscreator — COMMENTED** [review]
Looks reasonable to me.  :ship: :it:

**Review by muselesscreator — APPROVED** [review]

**Comment by mattcarter** [comment]
hey @nsprenkle!  This looks good, but is slipping into our past.  Can it be squashed & merged?
